docs/tutorials/scroll.md

```diff
@@ -0,0 +1,130 @@
+---
+section: docs
+layout: docs
+title: Scroll Snap
+---
+
+These examples shows the various scroll snap properties supported by atomic.
+
+### `scroll-behavior` construct
+
+The scroll-behavior CSS property sets the behavior for a scrolling box when scrolling is triggered by the navigation, `CSSOM` scrolling APIs or `SCROLL-SNAP`.
+
+```html
+<div class="Sb(s)">
+    <div>1</div>
+    <div>2</div>
+    <div>3</div>
+</div>
+```
+
+<div class="Sb(s) Sst(x_m) Ovx(s) Ovy(h) W(200px) H(200px) D(f) Mt(20px)">
+    <div class="Ssa(s) Miw(200px) H(200px) D(f) Ai(c) Jc(c) Fz(2em) Bgc(#add8e6) C(#ff)">1</div>
+    <div class="Ssa(s) Miw(200px) H(200px) D(f) Ai(c) Jc(c) Fz(2em) Bgc(#ccc)">2</div>
+    <div class="Ssa(s) Miw(200px) H(200px) D(f) Ai(c) Jc(c) Fz(2em) Bgc(#add8e6) C(#ff)">3</div>
+</div>
+
+
+### `scroll-snap-align` construct
+
+You can specify up to 2 values for this property, representing the block and inline axes, respectively. If you only give 1 value, that value will be used for both axes.
+
+```html
+<div class="Sb(s)">
+    <div class="Ssa(s)">1</div>
+    <div class="Ssa(c)">2</div>
+    <div class="Ssa(e)">3</div>
+</div>
+```
+
+<div class="Sb(s) Sst(x_m) Ovx(s) Ovy(h) W(200px) H(200px) D(f) Mt(20px)">
+    <div class="Ssa(s) Miw(160px) H(200px) D(f) Ai(c) Jc(c) Fz(2em) Bgc(#add8e6) C(#ff)">1</div>
+    <div class="Ssa(c) Miw(160px) H(200px) D(f) Ai(c) Jc(c) Fz(2em) Bgc(#ccc)">2</div>
+    <div class="Ssa(e) Miw(160px) H(200px) D(f) Ai(c) Jc(c) Fz(2em) Bgc(#add8e6) C(#ff)">3</div>
+</div>
+
+
+### `scroll-snap-type` construct
+
+The `scroll-snap-type` property specifies whether a scroll container is a scroll snap container, how strictly it snaps, and which axes are considered. If no strictness value is specified, proximity is assumed.
+
+
+```html
+<div class="Sb(s) Sst(x_p)">
```

Comment:
You should probably explain the `x_p` value in a paragraph above and perhaps link off to the MDN docs.